src/Artifacts.jl

```diff
@@ -45,7 +45,9 @@ current set of depot paths and the current artifact directory override via the m
 """
 function artifacts_dirs(args...)
     if ARTIFACTS_DIR_OVERRIDE[] === nothing
-        return [abspath(depot, "artifacts", args...) for depot in depots()]
+        depot_artifacts = [abspath(depot, "artifacts", args...) for depot in depots()] # search all depots
+        project_artifacts = [abspath(dirname(Base.active_project()), "artifacts", args...)] # search the active project
```

Comment:
Can `active_project()` be `nothing` in some circumstances?

Reply:
I'm not sure, but it can't hurt to be prepared for that case.

Reply:
Technically, `Base.ACTIVE_PROJECT[]` can be `nothing`.

Reply:
But what about `active_project()`. I think yes, but would need to be looked at.

Reply:
Alright, so now we check to make sure that `active_project()` is not `nothing` and `active_project()` is not empty.